pypesto/optimize/ess/ess.py

```diff
@@ -167,6 +171,7 @@ def _initialize(self):
         self.exit_flag: ESSExitFlag = ESSExitFlag.DID_NOT_RUN
         self.evaluator: Optional[FunctionEvaluator] = None
         self.starttime: Optional[float] = None
+        self.history: MemoryHistory = MemoryHistory()
```

Comment:
Do I see correctly, that History is now mandatory? Would at least set a flag or something to enable/disable history.

Reply:
It is tracked automatically, yes. The current history is usually quite short and I don't see a reason to not collect it. I see that as some basic diagnostic tool with low cost that should always be there.

At some point, there might be some option to track additional values for ensemble purposes which should not be done by default.